File: DayFour/JavaScript/AfterBreak/TaskTen.js

const prompt = require("prompt-sync")()

    let count = 1;
    let sum = 0;
    let counter = 0;

    for(; count < 11; count++){
        let score = Number(prompt("enter a number: "));
       if(score >= 0){
            sum += score;
            counter++;
        }
    }
    let average = sum/counter
   console.log(`${average}`);

File: DayFour/Java/AfterBreak/TaskTen.java
import java.util.Scanner;


public class TaskNine{

public static void main(String...ugo){
    Scanner scanner = new Scanner(System.in);
    int count = 1;
    int sum = 0;
    int counter = 0;

    System.out.println("enter a number: ");
    for(; count < 11; count++){
        int score = scanner.nextInt();

        if(score >= 0)
            sum += score;
            counter++;
    }
    int average = sum/counter;
    System.out.printf("sum: %d",average);

}


}
